File: 231214/특정 숫자 도달하기/reaching-specific-number.cpp
#include <iostream>
using namespace std;
int main() {
    int arr[10] = {};
    int sum = 0, cnt = 0;
    for (int i = 0; i < 10; i++) {
        cin >> arr[i];
    }

    for (int i = 0; i < 10; i++) {
        if (arr[i] >= 250) break;
        else {
            sum += arr[i];
            cnt++;
        }
    }
    cout << sum << " " << (double)sum / cnt;
    // 여기에 코드를 작성해주세요.
    return 0;
}

File: 250203/특정 숫자 도달하기/reaching-specific-number.py
arr = list(map(int, input().split()))
cal_arr = []

for el in arr:
    if el >= 250:
        break
    cal_arr.append(el)

sum = 0;

for el in cal_arr:
    sum += el;

print(sum, f"{sum / len(cal_arr):.1f}")
    
